cmd/wfx-viewer/cmd.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+/*
+ * SPDX-FileCopyrightText: 2024 Siemens AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Author: Michael Adler <michael.adler@siemens.com>
+ */
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/Southclaws/fault"
+	mcobra "github.com/muesli/mango-cobra"
+	"github.com/muesli/roff"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/siemens/wfx/cmd/wfx-viewer/output"
+	"github.com/siemens/wfx/cmd/wfx/metadata"
+	"github.com/siemens/wfx/generated/model"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	outputFlag       = "output"
+	outputFormatFlag = "output-format"
+)
+
+func init() {
+	rootCmd.Version = metadata.Version
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "man",
+		Short: "Generate man page and exit",
+		Run: func(*cobra.Command, []string) {
+			manPage, err := mcobra.NewManPage(1, rootCmd)
+			if err != nil {
+				log.Fatal().Err(err).Msg("Failed to generate man page")
+			}
+			manPage = manPage.WithSection("Copyright", "(C) 2023 Siemens AG.\n"+
```

Comment:
Hm, shouldn't this be 2024?

Reply:
Good question. I'm not a lawyer, but at least in US law, it's enough to note the first date of publication, see [section 401b2](https://www.copyright.gov/title17/92chap4.html)